user-config.yaml

```diff
@@ -0,0 +1,45 @@
+service:
```

Comment:
should this file be called `user-config.yaml`? It's ES-specific.

Reply:
No, if the user provides their own config, this will be used instead of `es-user-config.yaml`.

Reply:
is this file meant to be an example config?

Reply:
?